packages/web/src/components/tipping/feed-tip-tile/FeedTipTile.tsx

```diff
@@ -218,15 +248,41 @@ export const FeedTipTile = () => {
     }
   }, [dispatch, usersMap, tipToDisplay])
 
+  const handleResize = () => {
```

Comment:
We have a handy dandy `useMeasure` hook we could potentially use instead of attaching to the window:

https://github.com/streamich/react-use/blob/master/docs/useMeasure.md

might make some of this easier

Reply:
Oo nice